src/inc/gcinfoencoder.h

```diff
@@ -110,8 +110,12 @@ struct GcInfoSize
     size_t NumRanges;
     size_t NumRegs;
     size_t NumStack;
+    size_t NumUntracked;
```

Comment:
@swaroop-sridhar @jkotas Updated accordingly. I adjusted the order of the new fields to align to the existing code as well.